libfwupdplugin/meson.build

```diff
@@ -8,6 +8,33 @@ fwupdplugin_version_h = configure_file(
   configuration: conf
 )
 
+fwupdplugin_struct = []
+fwupdplugin_struct_srcs = []
+foreach generated: [
+  'fu-acpi-table',  # fuzzing
+  'fu-self-test',   # fuzzing
+]
```

Comment:
thinking through how plugins will probably want to be providing these as well, probably need to make the array additive.

Reply:
I've switched to using a meson generator and it's much much nicer.